src/pages/SignIn/components/FacebookSignInButton.jsx

```diff
@@ -0,0 +1,26 @@
+import { FacebookAuthProvider, signInWithPopup } from 'firebase/auth';
+import { auth } from '../../../utils/firebase';
+import { Button } from '@nextui-org/react';
+import Facebook from '../../../assets/facebook.svg';
+
+const FacebookSignInButton = () => {
+  const userSignIn = async () => {
+    const provider = new FacebookAuthProvider();
+    try {
+      const result = await signInWithPopup(auth, provider);
```

Comment:
when i click the button, the popup says the app is deactivated

Reply:
updated the App ID and private code to Meta dev site. should work now